src/contexts/RDKitContext.tsx

```diff
@@ -5,12 +5,16 @@ export interface RDKitContextValue {
   RDKit: RDKitModule | null;
 }
 
+export type RDKitProviderProps = PropsWithChildren<{
+  rdkitInstance?: RDKitModule;
+}>;
+
 // force default context to be undefined, to check if package users have wrapped it with the required provider
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const RDKitContext = React.createContext<RDKitContextValue>(undefined as any);
 
-export const RDKitProvider: React.FC<PropsWithChildren> = ({ children }) => {
-  const [RDKit, setRDKit] = useState<RDKitModule | null>(null);
+export const RDKitProvider: React.FC<RDKitProviderProps> = ({ rdkitInstance, children }) => {
+  const [RDKit, setRDKit] = useState(rdkitInstance ?? null);
```

Comment:
@atfera  We should add a check to not call `initRDKitModule` if we have passed an already loaded rdkit instance to the context.

Reply:
Ah my bad, I added a check but I don't know why it's not in the commit.